.github/workflows/CI-CHGLOG.yml

```
@@ -0,0 +1,369 @@
+    steps:
+      - name: Download ChangeLog Artifact
+        id: download
+        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093 # v4.3.0
+        with:
+          path: '${{ runner.temp }}/multicast-chglog-${{ needs.check_build.outputs.sha }}'
+          artifact-ids: ${{ needs.CHGLOG.outputs.artifact-id }}
+          repository: reactive-firewall/multicast
+      - name: "Move chglog into place"
+        id: load_chglog_info
+        if: ${{ always() }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)